.github/workflows/performance.yml

```diff
@@ -0,0 +1,27 @@
+name: "Performance"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+env:
+  fail-fast: true
+
+jobs:
+  performance-tests:
+    name: "Tests for the performance testing the PDF parsing"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php:
+          - "7.4"
```

Comment:
In your opinion, is it worth testing performance with all supported PHP versions? If so, please add 7.2, 7..3 and 8.0 here.

Reply:
As the issue was currently with the code, not with the PHP version, then I don't think running it with different PHP versions helps a lot right now (rather it will make the testing unnecessarily slower).

Reply:
Fair point.